src/ucp/core/ucp_am.c

```diff
@@ -1347,11 +1387,20 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_am_long_middle_handler,
     ucp_am_mid_hdr_t *mid_hdr  = am_data;
     ucp_ep_h ep                = ucp_worker_get_ep_by_id(worker,
                                                          mid_hdr->ep_id);
-    ucp_ep_ext_proto_t *ep_ext = ucp_ep_ext_proto(ep);
     uint64_t msg_id            = mid_hdr->msg_id;
     ucp_recv_desc_t *mid_rdesc = NULL, *first_rdesc = NULL;
+    ucp_ep_ext_proto_t *ep_ext;
     ucs_status_t status;
 
+    if (ucs_unlikely(!ucp_am_ep_is_valid(ep))) {
+        /* ep is closed or failed, drop the data */
+        ucs_trace_data("worker %p: drop middle AM fragment on closed/failed ep %p",
+                       worker, ep);
+        return UCS_OK;
+    }
+
+    ep_ext      = ucp_ep_ext_proto(ep);
+
```

Comment:
empty line?

Reply:
remove the empty line or don't need the alignment by `=`

Reply:
i'll remove the line